### PR TITLE
Fix for issue 9362 part 2 - Move location of the pin button of the global search dialog to the top

### PR DESCRIPTION
**Pull request by ssrpro**
Fixes part 2 of https://github.com/JabRef/jabref/issues/9362

test3
The location of the pin button of the Global Search dialog is now moved to the top right corner instead of the bottom left corner.

```[tasklist]
### Compulsory checks
- [x] Change in `CHANGELOG.md` described in a way that is understandable for the average user (if applicable)
- [ ] Tests created for changes (if applicable)
- [x] Manually tested changed features in running JabRef (always required)
- [x] Screenshots added in PR description (for UI changes)
- [x] [Checked developer's documentation](https://devdocs.jabref.org/): Is the information available and up to date? If not, I outlined it in this pull request.
- [x] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not, I created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, I submitted a pull request to the documentation repository.
```

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -92,6 +92,7 @@ Note that this project **does not** adhere to [Semantic Versioning](http://semve
 - We fixed an issue where an exception occured when manually downloading a file from an URL in the entry editor. [#9521](https://github.com/JabRef/jabref/issues/9521)
 - We fixed an issue with open office csv file formatting where commas in the abstract field where not escaped. [#9087][https://github.com/JabRef/jabref/issues/9087]
 - We fixed an issue with deleting groups where subgroups different from the selected group were deleted. [#9281][https://github.com/JabRef/jabref/issues/9281]
+- We fixed an issue where the pin button on the Global Search dialog was located on the wrong location. [#9362](https://github.com/JabRef/jabref/issues/9362)
```
Comment (Siedlerchr) [inline CHANGELOG.md:95]:
```suggestion
- We fixed an issue where the pin button on the Global Search dialog was located at the bottom and not at the top. [#9362](https://github.com/JabRef/jabref/issues/9362)
```

**Review by Siedlerchr — APPROVED** [review]

**Comment by calixtus** [comment]
Mhh... looks still odd. Now we have an empty line at the top of the window...
I suggest rather to use an `AnchorPane` to draw the pin over the table column names, like in `CitationKeyPatternTab.fxml`:
```html
    <AnchorPane>
        <CitationKeyPatternPanel fx:id="bibtexKeyPatternTable" AnchorPane.leftAnchor="0.0" AnchorPane.rightAnchor="0.0" prefHeight="180.0"/>
        <Button text="%Reset All" onAction="#resetAllKeyPatterns" AnchorPane.rightAnchor="0.0" AnchorPane.topAnchor="0.0"/>
    </AnchorPane>
```

**Comment by koppor** [comment]
@ssrpro I commited the CHANGELOG.md suggestion by Christoph. Please pull before you continue working on the PR.

**Comment by ssrpro** [comment]
It seems that putting an AnchorPane inside the SplitPane does not work properly when adjusting the split position.

Changing the code to this:
`
    <content>
        <SplitPane fx:id="container" orientation="VERTICAL">
            <AnchorPane fx:id="tablePane">
                <ToggleButton fx:id="keepOnTop" styleClass="icon-button,narrow" prefHeight="20.0" prefWidth="20.0" AnchorPane.rightAnchor="0.0" AnchorPane.topAnchor="0.0">
                    <graphic>
                        <JabRefIconView glyph="KEEP_ON_TOP"/>
                    </graphic>
                    <tooltip>
                        <Tooltip text="%Keep dialog always on top"/>
                    </tooltip>
                </ToggleButton>
            </AnchorPane>
        </SplitPane>
    </content>
`
produces the following UI
image
The split can not be moved at all and empty rows appear in the table view.

Adding a ScrollPane as an intermediate node gives the following code:
`
    <content>
        <SplitPane fx:id="container" orientation="VERTICAL">
            <ScrollPane>
                <AnchorPane fx:id="tablePane">
                    <ToggleButton fx:id="keepOnTop" styleClass="icon-button,narrow" prefHeight="20.0" prefWidth="20.0" AnchorPane.rightAnchor="0.0" AnchorPane.topAnchor="0.0">
                        <graphic>
                            <JabRefIconView glyph="KEEP_ON_TOP"/>
                        </graphic>
                        <tooltip>
                            <Tooltip text="%Keep dialog always on top"/>
                        </tooltip>
                    </ToggleButton>
                </AnchorPane>
            </ScrollPane>
        </SplitPane>
    </content>
`
And gives the following UI
image

This makes the split movable, but empty rows are still appearing.

So my question is, is it really feasible to put an ArchonPane inside a SplitPane without getting those issues and should I keep looking for a solution with this strategy, or is there another way to fix this problem al together?

**Comment by Siedlerchr** [comment]
I think then go with the first solution!

**Review by Siedlerchr — APPROVED** [review]

**Review by Siedlerchr — APPROVED** [review]

**Pull request merged**